spec/support/fixtures/vcr_cassettes/coll_8_api/operations/invalid_recipient_postcode_error.yml

```diff
@@ -0,0 +1,50 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://ship-api-uat.drop2shop.ie/api/shipment
+    body:
+      encoding: UTF-8
+      string: '{"account":{"accountCode":"<ACCOUNT_REF>"},"serviceType":"drop2me","goodsDescription":"Sample
+        Signature Required","shippingDate":"2020-09-03","numberOfPackages":1,"references":[{"referenceType":"customerReference","referenceValue":"SH-Me
+        0107-001"}],"packages":[{"packagingType":"Box","weight":1}],"signatureServices":[{"service":"SignatureRequired"}],"addresses":[{"addressLine1":"Shippers
+        Lane","city":"London","postCode":"EC2R","countryCode":"GBR","country":"Great
+        Britain","addressType":"shipFrom","addressLocationType":"business","contact":{"contactName":"shipper
+        contact","email":"shipper@business.com","mobileNumber":"+4470000000"}},{"addressLine1":"Merrion
+        St Upper","city":"Dublin 2","postCode":"D02 R583","state":"Dublin","country":"Ireland","addressType":"shipTo","contact":{"contactName":"consignee
+        name","email":"consignee@receiver.com","mobileNumber":"+353870000000"},"postcode":"DB2"}]}'
+    headers:
+      User-Agent:
+      - Faraday v1.0.1
+      Authorization:
+      - Bearer <BEARER_TOKEN>
+      Content-Type:
+      - application/json
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+      Accept:
+      - "*/*"
+  response:
+    status:
+      code: 400
+      message: Bad Request
+    headers:
+      Content-Length:
+      - '61'
+      Content-Type:
+      - application/json; charset=utf-8
+      Server:
+      - Microsoft-IIS/10.0
+      Request-Context:
+      - appId=cid-v1:85e0e9e3-0c9e-457e-a258-1b3cf8258e9a
+      X-Powered-By:
+      - ASP.NET
+      Set-Cookie:
+      - ARRAffinity=9360bb9059cba4f95253150408057590cb934bd7d9b6e4f77657b48444b6bd67;Path=/;HttpOnly;Domain=ship-api-uat.drop2shop.ie
+      Date:
+      - Tue, 01 Sep 2020 11:05:37 GMT
+    body:
+      encoding: UTF-8
+      string: '{"errors":[{"message":"Recipient''s Postcode is incorrect."}]}'
```

Comment:
Error messages could be extracted; this seems like a valid JSON and is consistent with other error responses.

Reply:
I spoke too soon...